SWENT-GRP5/SOLVIT
# Remove unused buttons and deprecated screens

**Description:**
- Removed all unused buttons in the following files:
  - `MessageBox.kt`
  - `SignUpChooseProfile.kt`
  - `RequestsFeed.kt`
  - `SignIn.kt`
- Deleted the `ForgotPassword` screen since it was not implemented.
- Adapted and updated the relevant tests to reflect these changes, ensuring consistency with the updated codebase.

**Related Issues:**
- Fixes #425

**How to Test:**
- To test this pull request, simply navigate through the entire app and verify that there are no longer any non-functional buttons.
- Note: A few buttons have been left in place intentionally for future implementation. These buttons are currently non-functional but serve as placeholders for upcoming features.
